common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -179,9 +180,15 @@ private void renderTexturedCount(PoseStack poseStack, int x, int y, int emeralds
                     Texture.EMERALD_COUNT_BACKGROUND.width(),
                     Texture.EMERALD_COUNT_BACKGROUND.height());
 
-            McUtils.mc()
-                    .getItemRenderer()
-                    .renderGuiItem(poseStack, EmeraldUnits.values()[i].getItemStack(), renderX + 6, renderY + 6);
+            if (EmeraldUnits.values()[i].getSymbol().equals("stx")) { // Make stx not look like normal LE
+                RenderUtils.drawTexturedRect(
```

Comment:
Oh. I am kind of stupid here. We did not have textures for the other values, we just drew the vanilla items. 🤦 

I thought we would be able to unify stx with the other emerald denominations.

Eh... I am terribly sorry for giving you extra work, but can you please revert back to your original implementation? That was much better; I just did not read it properly. :-(